.jscpd.json

```diff
@@ -3,7 +3,7 @@
   "maxSize": 0,
   "threshold": 0,
   "reporters": ["consoleFull", "console"],
-  "ignore": ["**/.git", "**/.venv"],
+  "ignore": ["**/.git", "**/.venv", "workflow-templates"],
```

Comment:
Is it correct that you added `"workflow-templates"` in order to avoid triggering jscpd because of the repeated hash now that slack-templates is pinned? Is there any practical way to get around the jscpd failure without ignoring the entire file?

Reply:
Yeah, the hashes increased the lengths of the clones beyond the maximum allowed 49 characters. I could ignore only specific snippets of each file, but the issue may recur in the future if we make other modifications since the workflow templates are fundamentally so similar to the workflows. I haven't been able to come up with a more elegant solution, but I could believe there is something I missed.

Reply:
Fair enough! Thanks for the explanation.